src/ReactLeafletMap/index.js

```diff
@@ -87,20 +87,29 @@ class ReactLeafletMap extends React.PureComponent {
     }
 
     onViewportChange(viewport) {
-        const center = {
-            lat: viewport.center[0],
-            lon: viewport.center[1]
-        };
-
-        const boxRange = viewUtils.getBoxRangeFromZoomLevel(viewport.zoom);
+        const change = {};
+        // viewport.center could be undefined
+        if(viewport && viewport.hasOwnProperty('center') && viewport.center) {
```

Comment:
```suggestion
        if(viewport && viewport.center) {
```

Reply:
As in comment viewport.center can be undefined

Reply:
Then viewport && viewport.center is falsey

Reply:
true